extensions/amp-ad-network-adsense-impl/0.1/test/test-adsense-a4a-config.js

```diff
@@ -63,6 +64,19 @@ describe('adsense-a4a-config', () => {
       expect(adsenseIsA4AEnabled(mockWin, elem)).to.be.false;
     });
 
+    it('should not enable a4a when on a non-Google AMP cache', () => {
+      mockWin.location = parseUrl(
+          'https://amp.cloudflare.com/some/path/to/content.html');
+      sandbox.stub(
+          urls, 'cdnProxyRegex',
```

Comment:
this works in practice but shouldn't be done as modules are not really objects and can't be stubbed